packages/react-components/src/components/table-output-component.tsx

```diff
@@ -83,13 +84,13 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
                     this.fetchColumns = false;
                     await this.init();
                 }
-                const rowsThisPage = await this.fetchSearchTableLines(params.startRow, params.endRow - params.startRow);
+                const rowsThisPage = await this.fetchTableLines(params.startRow, params.endRow - params.startRow);
                 for (let i = 0; i < rowsThisPage.length; i++) {
                     const item = rowsThisPage[i];
                     const itemCopy = cloneDeep(item);
                     rowsThisPage[i] = itemCopy;
                 }
-                params.successCallback(rowsThisPage, this.props.nbEvents);
+                params.successCallback(rowsThisPage, this.tableSize);
```

Comment:
Can we use rowThisPage.size instead of introducing the new property this.tableSize?

Reply:
The fetchTableLines fetches by a chunck of 200 rows at a time so the rowsThisPage.length will be 200. However the table may have more lines than 200. So unfortunately, your change request is not applicable.

Reply:
Sounds good